src/entities/ProductSchema.ts

```diff
@@ -0,0 +1,31 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+export const ProductSchema = z.object({
+  name: z
+    .string()
+    .min(3, { message: "Name must be at least 3 characters long." })
+    .max(50, { message: "Name must be at most 50 characters long." }),
+  category: z.string().uuid({ message: "Category must be a valid UUID." }),
```

Comment:
I don't think that we should check for the uuid because our users do not know about uuids.

Reply:
Well, the category field in the form is a select field that has the values set to the product categories' ids retrieved from the backend. Technically, that field should always be a uuid.

Anyways, I can remove that check if necessary.

Reply:
Modified the message as to be more suggestive to the user:
"Category cannot be found."